.github/workflows/test.yml

```diff
@@ -26,37 +26,27 @@ jobs:
       fail-fast: false
       matrix:
         include:
-        - name: Test suite with py37-windows-64
-          python: '3.7'
-          arch: x64
-          os: windows-latest
-          toxenv: py37
-        - name: Test suite with py38-ubuntu
+        # Make sure to run mypyc compiled unit tests for both
+        # the oldest and newest supported Python versions
+        - name: Test suite with py38-ubuntu, mypyc-compiled
           python: '3.8'
           arch: x64
           os: ubuntu-latest
           toxenv: py
           tox_extra_args: "-n 2"
-        - name: Test suite with py39-ubuntu
-          python: '3.9'
-          arch: x64
-          os: ubuntu-latest
-          toxenv: py
-          tox_extra_args: "-n 2"
-        - name: Test suite with py37-ubuntu, mypyc-compiled
-          python: '3.7'
+          test_mypyc: true
+        - name: Test suite with py38-windows-64
+          python: '3.8'
           arch: x64
-          os: ubuntu-latest
-          toxenv: py
+          os: windows-latest
+          toxenv: py38
           tox_extra_args: "-n 2"
```

Comment:
I wonder if my previous suggestion might have been bag; maybe this might be the thing that's breaking things? Maybe there's a reason that the windows py37 job was the only job that didn't have `-n 2` passed to it...

Reply:
Didn't work :-/

Reply:
yeah, i saw :(